tests/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:sid
+RUN apt-get update -q && apt-get install -y \
```

Comment:
Maybe most of it can be replaces with using `build-dep umockdev`?

Reply:
I have sent couple of patches to umockdev adding support for thunderbolt, it may take some time to get package ready.

Reply:
although using build-dep might be a good idea

Reply:
This is answer for why not to use the package found in Debian repos (i.e. why not *`install`*` umockdev`), but I assume most of the list of the packages here is for build dependencies for umockdev, and thus can be pulled by `apt` together just by asking for *`build-dep`*` umockdev` instead of mentioning them one-by-one (and update it if the package structure changes for one of the dependencies).

Reply:
(Oh, you added another comment before I finished mine :)
For some reason, GitHub didn't display it on time.)

Reply:
I tested on my local host and found that "sudo apt-get -s build-dep umockdev" would install more packages then needed.

Reply:
:(